Ports/AvailablePorts.md

```diff
@@ -198,6 +198,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`tuxracer`](tuxracer/)                | Tux Racer                                                       | 0.61                     | http://tuxracer.sourceforge.net/                                               |
 | [`tcl`](tcl/)                          | Tcl                                                             | 8.6.11                   | https://www.tcl-lang.org/                                                      |
 | [`termcap`](termcap/)                  | GNU termcap                                                     | 1.3.1                    | https://www.gnu.org/software/termutils/                                        |
+| [`powdertoy`](powdertoy/)              | The Powder Toy                                                  | 96.2.350                 | https://powdertoy.co.uk/                                                       |
```

Comment:
Please keep this sorted alphabetically :)

Reply:
Oops

Reply:
Apparently SDLPoP is out of order too